<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonCreator</class>
 <widget class="QDialog" name="JsonCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Work</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="createNewBtn">
        <property name="text">
         <string>Create New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addNewLine">
        <property name="text">
         <string>Create New Test Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveNewToFile">
        <property name="text">
         <string>Save New Line to File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadExFileBtn">
        <property name="text">
         <string>Load File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="doneBtn">
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditFileLoad"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
